src/components/button/button.css

```diff
@@ -0,0 +1,67 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.interactivity {
+  cursor: pointer;
+}
+
+.typography {
+  font-family: var(--mavenlink-type-font-family);
+  font-size: var(--mavenlink-type-button-font-size);
+  line-height: var(--spacing-x-large);
+}
+
+.button {
```

Comment:
Question for design: is there a "active" state for the a normal button? legacy styleguide has some inset styling.

Reply:
Question for design: are we ensuring our components have the appropriate `box-sizing: border-box`?
Question for design + engineering: what other "self-contained" styling do we need?

Reply:
Global resets like zeroing out margins. Also look for helpers e.g. we had a antialiasing thing and if other stuff it should live besides that. But from what we did in self contained components I'd say yes, we want to use `box-sizing`. To my mind, Design probably doesn't care as that's an implementation detail so long as the component is compliant with the spacing spec and any other similar design specs.